viam-cartographer/src/carto_facade/carto_facade.cc

```diff
@@ -41,6 +41,14 @@ std::string to_std_string(bstring b_str) {
     return std_str;
 }
 
+bstring to_bstring(std::string str) {
```

Comment:
Just checking, does the better string library not have something like this?

Reply:
Unfortunately not given that better string is a C library. There is a C++ wrapper but it doesn't provide conversion functions to / from std::string.

Reply:
Got it, it makes sense that it wouldn't since it's a C library, and the C++ wrapper (somewhat surprisingly) doesn't provide those helpers.